configure.ac

```diff
@@ -284,6 +286,9 @@ AC_MSG_CHECKING([what semaphore type to use]);
 AS_IF([test "x$have_mach" = "xtrue"],
   [AC_DEFINE(USE_MACH_SEM, 1, [Define to use Mach semaphores])
     AC_MSG_RESULT([Mach semaphores])],
+  [test "x$have_futex" = "xtrue"],
+  [AC_DEFINE(USE_FUTEX_SEM, 1, [Define to use Futex semaphores])
```

Comment:
What I mean is that I can tell you'll want to use FUTEX for other things, so I wouldn't make this scoped to "semaphores" only.